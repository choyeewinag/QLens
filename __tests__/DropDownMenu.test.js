import React, { useState, useEffect } from 'react';
import renderer from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow';
import DropDownMenu from '../src/Components/DropDownMenu';


//Basic Test with React-test-renderer
it('renders correctly react-test-renderer', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<DropDownMenu />);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});