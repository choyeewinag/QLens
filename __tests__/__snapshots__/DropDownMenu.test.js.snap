// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly react-test-renderer 1`] = `
<div
  className="dropDown"
>
  <div />
  <div
    className="addSchemaBtn"
  >
    <button
      className="AddSelectedSchemasButton"
    >
      <span>
        Add Selected Schemas
      </span>
    </button>
  </div>
</div>
`;
